scripts/install-wheels.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+cd `dirname $BASH_SOURCE` && cd ..
+
+# Install "wheel" archives of the requirements for running the test suite.
+# http://pip.readthedocs.org/en/latest/reference/pip_wheel.html
+# This runs in Travis to install pre-built binary packages, which
+# means the builds are faster and more reliable.
+sudo pip install --upgrade pip
+sudo pip install wheel
+
+WHEELS_ARGS=""  # Get wheels from the internet by default
+
+#if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
```

Comment:
Let's make this only applicable on TravisCI. Use the [TRAVIS](http://stackoverflow.com/a/12989131/161278) environment variable. So instead of Linux, let's run this script for TravisCI servers.